views/components/ChartList.jsx

```diff
@@ -1,16 +1,16 @@
 import React from 'react'
-import Table from '@material-ui/core/Table'
-import TableBody from '@material-ui/core/TableBody'
-import TableCell from '@material-ui/core/TableCell'
-import TableHead from '@material-ui/core/TableHead'
-import TableRow from '@material-ui/core/TableRow'
-import Typography from '@material-ui/core/Typography'
-import Button from '@material-ui/core/Button'
-import Link from '@material-ui/core/Link'
-import Delete from '@material-ui/icons/Delete'
-import Edit from '@material-ui/icons/Edit'
-import Share from '@material-ui/icons/Share'
-import PlaylistAdd from '@material-ui/icons/PlaylistAdd'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import Typography from '@mui/material/Typography'
+import Button from '@mui/material/Button'
+import Link from '@mui/material/Link'
+import Delete from '@mui/icons-material/Delete'
+import Edit from '@mui/icons-material/Edit'
+import Share from '@mui/icons-material/Share'
+import PlaylistAdd from '@mui/icons-material/PlaylistAdd'
```

Comment:
I think we can make this 2 imports, 1 for the components and 1 for the icons.

Reply:
[6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)